index.js

```diff
@@ -67,6 +67,35 @@ function toggleColor(element) {
     }
 }
 
+const headerText = document.querySelector('h1#header')
+
+headerText.addEventListener('click', () => {
+    toggleColor(headerText)
+})
+
+
+
+
 /***** Deliverable 2 *****/
```

Comment:
You are on the right track in finding the likeButton and the likes `p` tag. Let's lay this problem out and look at how we might fully execute it such that we're only updating the likes in each `p` tag...

1. First find the like button element on the page
2. Then add an event listener to the like button
3. We are listening for a `click` event, and we will pass a callback function to be invoked when the event fires.
4. Then we create the function `addNewLike` which will handle all the functionality.

- The `addNewLike` function has a few steps:
1. find the `p` tag where the current number of likes is stored on the page
2. extract the number out of the string, make sure to convert it to a number with `parseInt()`
3. add 1 to that number.
4. Create a new string with the new number
5. update the `p` tag to have the newly created string as its `innerText`.

```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
- Or with ES6 notation
```javascript
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```